web_responsive/views/web.xml

```diff
@@ -11,8 +11,8 @@
               inherit_id="web.webclient_bootstrap"
               name="App Drawer - Web Client"
               >
-        <xpath expr="//*[hasclass('o_main')]" position="attributes">
-            <attribute name="t-attf-class">o_main o_chatter_position_{{ request.env.user.chatter_position or 'normal' }}</attribute>
+        <xpath expr="//t[@t-set='body_classname']" position="attributes">
+            <attribute name="t-value">'o_web_client o_chatter_position_' + (request.env.user.chatter_position or 'normal')</attribute>
```

Comment:
Why this change? It's a bit weird.

Reply:
This change was made because 'o_document_viewer' is out from '.o_main'.